app/index.html

```diff
@@ -20,7 +18,7 @@
 		}
 	</script>
 
-	<main>Loading...</main>
+	<main></main>
 
-	<script type='module' src='./js/index.js'></script>
+	<script type="module" src="./js/index.js"></script>
```

Comment:
```suggestion
	<script async type="module" src="./js/index.js"></script>
```
While not required, we recently learned making this `async` is probably more correct and may help avoid some weird race conditions in FF.